### PR TITLE
refacto(front/login/a11y): refactoring, improve a11y in login page part 1

### PR DESCRIPTION
**Pull request by LiquidITGuy**
part 1 of improving accessibility and refactoring to lighten the code. e2e testing have been added to prevent regressions. unit and integration testing will be added on the nexts pull requests when components will be more decouplated

First of all, thank you for your contribution! 😄

Please make sure you have read [contribution guidelines](https://github.com/caprover/caprover/blob/master/CONTRIBUTING.md#before-contributing). 

### Most important items

- Make sure to communicate your proposed changes ob our Slack channel beforehand.
- Large PRs (50+ lines of code) will get rejected due to increased difficulty for review - unless they have been communicated beforehand with project maintainers.
- **Refactoring work will get rejected** unless it's been communicated with project's maintainers **beforehand**. There is a thousand ways to write the same code. Every time a code is changed, there is a potential for a new bug. We don't want to refactor the code just for the sake of refactoring.

These rules are strictly enforced to make sure that we can maintain the project moving forward.

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (githubsaturn) [inline cypress/fixtures/example.json:1]: is this file needed?

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```
Comment (githubsaturn) [inline cypress/support/commands.ts:1]: is this file needed?

File: cypress/support/e2e.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/e2e.ts is processed and
```
Comment (githubsaturn) [inline cypress/support/e2e.ts:2]: Same with this file, is it needed?

File: package.json [hunk 4]
```diff
@@ -67,15 +69,16 @@
         "@types/react-router": "^5.1.11",
         "@types/react-router-dom": "^5.1.7",
         "@types/utf8": "^2.1.6",
-        "tslint": "^6.1.3",
-        "typescript": "^4.1.5",
+        "cypress": "^12.5.1",
```
Comment (githubsaturn) [inline package.json:72]: Had never heard of Cypress! Nice framework!

**Review by githubsaturn — COMMENTED** [review]
Thanks! Code improvements are great! Will chat more about the testing framework over Slack.

File: cypress/fixtures/example.json (hunk 1 above)
Comment (LiquidITGuy) [reply cypress/fixtures/example.json]: file removed

File: cypress/support/commands.ts (hunk 2 above)
Comment (LiquidITGuy) [reply cypress/support/commands.ts]: code updated and file removed

File: cypress/support/e2e.ts (hunk 3 above)
Comment (LiquidITGuy) [reply cypress/support/e2e.ts]: this file is needed to run the tests.
I remove useless imports and add a devnote to explain the goal of the file

**Comment by LiquidITGuy** [comment]
To pass the automate testing, doing a yarn install with a x86 processor would fix all of that and commit the yarn.lock file (I use an arm processor who can fix the errors but can add some compatibility behaviour to some lib).
(if you want to run e2e testing do not forget to start the front before)

**Review by githubsaturn — COMMENTED** [review]
Just one minor comment. It's a typo I assume.
